src/java.base/share/classes/java/lang/Process.java

```diff
@@ -99,7 +116,10 @@
  * process and for the communication streams between them.
  * The resources to control the process and for communication between the processes are retained
  * until there are no longer any references to the Process or the input, error, and output streams
- * or readers, or they have been closed.
+ * or readers, or they have been closed. The Process {@linkplain Process#close close} method closes
+ * all the streams and terminates the process to release the resources. Using try-with-resources to
+ * {@linkplain ProcessBuilder#start()} the process can ensure the process
```

Comment:
This link looks weird in the generated documentation:

> Using try-with-resources to ProcessBuilder.start() the process can ensure the process is terminated when the try-with-resources block exits.

I suggest fixing the text to just "start" and optionally removing the link as that method is already linked to from this same doc comment:


```suggestion
 * {@linkplain ProcessBuilder#start() start} the process can ensure the process
```